packages/material-ui-codemod/README.md

```diff
@@ -28,34 +28,130 @@ Options:
                                                 [boolean] [default: false]
   --print    print transformed files to stdout, useful for development
                                                 [boolean] [default: false]
+  --jscodeshift                                  [string] [default: false]
 
 Examples:
   npx @material-ui/codemod@next v4.0.0/theme-spacing-api src
   npx @material-ui/codemod@next v5.0.0/component-rename-prop src --
   --component=Grid --from=prop --to=newProp
 ```
 
+#### jscodeshift options
+
+To pass more options directly to jscodeshift, use `--jscodeshift="..."`. For example:
+
+```sh
+npx @material-ui/codemod@next --jscodeshift="--run-in-band --verbose=2"
+```
+
+See all available options [here](https://github.com/facebook/jscodeshift#usage-cli).
+
+#### Recast Options
+
+Options to [recast](https://github.com/benjamn/recast)'s printer can be provided
+through jscodeshift's `printOptions` command line argument
+
+```sh
+npx @material-ui/codemod@next <transform> <path> --jscodeshift="--printOptions='{\"quote\":\"double\"}'"
+```
+
 ## Included scripts
 
 ### v5.0.0
 
-#### `component-rename-prop`
+#### 🚀 `preset-safe`
 
-A generic codemod to rename any component prop.
+A combination of all important transformers for migrating v4 to v5. ⚠️ This mod is not idempotent (`theme-breakpoints`) and should be run only once.
 
-```diff
--<Component prop="value" />
--<Component prop />
-+<Component newProp="value" />
-+<Component newProp />
+```sh
+npx @material-ui/codemod@next v5.0.0/preset-safe <path|folder>
 ```
 
-<!-- #default-branch-switch -->
+The list includes
+
+- [@material-ui/codemod](#material-uicodemod)
+  - [Setup & run](#setup--run)
+    - [jscodeshift options](#jscodeshift-options)
+    - [Recast Options](#recast-options)
+  - [Included scripts](#included-scripts)
+    - [v5.0.0](#v500)
+      - [🚀 `preset-safe`](#-preset-safe)
+      - [`adapter-v4`](#adapter-v4)
+      - [`autocomplete-rename-closeicon`](#autocomplete-rename-closeicon)
+      - [`avatar-circle-circular`](#avatar-circle-circular)
```

Comment:
This TOCs is weird. What is it doing here?

Reply:
Fixed, it is the auto-formatting when I saved the file 😅